test/unit/utils/url.test.js

```diff
@@ -1,6 +1,8 @@
 import document from 'global/document';
 import window from 'global/window';
 import * as Url from '../../../src/js/utils/url.js';
+import proxyquireify from 'proxyquireify';
+const proxyquire = proxyquireify(require);
```

Comment:
Proxyquireify uses a module called `merge-descriptors` that is currently throwing some warnings in chrome. I'll open an issue against it so we can get that closed out.
![](https://cldup.com/FUj72ZuM40.png)